R/print.data.table.R

```diff
@@ -164,3 +186,47 @@ shouldPrint = function(x) {
 #   as opposed to printing a blank line, for excluding col.names per PR #1483
 cut_top = function(x) cat(capture.output(x)[-1L], sep = '\n')
 
+# to calculate widths of data.table for PR #4074
+nchar_width = function(x) {
+  each_width = nchar(x, type = "width")
```

Comment:
`apply(nchar(x, type='width'), 2L, max)` right? If so I'd drop this helper and just use

```
widths = apply(nchar(x, type='width'), 2L, max)
```

Reply:
Yep, that would do the same. Will replace.